tests/ocp/hwol/tests/hwol.go

```
@@ -0,0 +1,570 @@
+  ip=$(ip -4 -o addr show dev "$iface" 2>/dev/null | awk '{print $4}' | cut -d/ -f1)
+  [ -n "$ip" ] && break
+  sleep 1
+done
+[ -n "$ip" ] || { echo "no IPv4 on $iface"; exit 1; }
+exec iperf3 -s -B "$ip"
+`}
+}
+
+// waitForHwolResource waits until available openshift.io/hwolresource on the node
+// (allocatable minus non-terminated pod requests) is at least min.
+func waitForHwolResource(nodeName string, min int64, timeout time.Duration) error {
```

golangci-lint (2.12.2)

[error] param min has same name as predeclared identifier

(predeclared)